docs/release-testing.md

```diff
@@ -0,0 +1,115 @@
+## Generating and testing release bundles 
+
+More specific documentation for generating and testing bundles can be found [here](https://operator-framework.github.io/community-operators/testing-operators/).
+
+### Pre-requisites 
+Testing the bundle requires `opm`. To install opm, clone the operator-registry repository: 
+
+```
+git clone https://github.com/operator-framework/operator-registry
+```
+Change into the directory and run `make build`. This will generate an `opm` executable in the `operator-registry/bin/` directory. Add the bin to your path, or substitute `opm` in the following commands with the full path to your `opm` executable.
+
+### Generating the release bundle 
+Build and push the updated operator image to your quay.io repository for testing: 
+
+```sh
+$ export USERNAME=<quay-username>
+$ export VERSION=<release-version>
+docker build . -t quay.io/$USERNAME/koku-metrics-operator:$VERSION
+docker push quay.io/$USERNAME/koku-metrics-operator:$VERSION 
+```
+
+Update the release version at the top of the `Makefile` to match the release version of the operator: 
+
+```
+# Current Operator version
+VERSION ?= <release-version>
+```
+Run the following command to generate the bundle: 
+
+```
+make bundle DEFAULT_CHANNEL=alpha
+```
+This will generate a new `<release-version>` bundle inside of the `koku-metrics-operator` directory within the repository. 
+
+### Edit the bundle csv to point to the test image
+Search and replace `quay.io/project-koku/koku-metrics-operator:$VERSION` with `quay.io/$USERNAME/koku-metrics-operator:$VERSION` in the generated clusterserviceversion. 
+
+
+### Testing the release bundle 
+Once the release bundle has been generated, fork & clone the [community-operators repository](https://github.com/operator-framework/community-operators). Create a branch, and copy the generated bundle to the `community-operators/community-operators/koku-metrics-operator/` directory in your cloned fork. 
+
+For example, if the bundle was generated for a `1.0.0` release, the directory structure would look like the following: 
+
+```
+koku-metrics-operator/
+├── 0.9.0
+│   ├── manifests
+│   │   ├── koku-metrics-cfg.openshift.io_kokumetricsconfigs.yaml
+│   │   └── koku-metrics-operator.clusterserviceversion.yaml
+│   ├── metadata
+│   │   └── annotations.yaml
+│   └── Dockerfile
+├── 1.0.0
+│   ├── manifests
+│   │   ├── koku-metrics-cfg.v1.0.0.openshift.io_kokumetricsconfigs.yaml
+│   │   └── koku-metrics-operator.v1.0.0.clusterserviceversion.yaml
+│   ├── metadata
+│   │   └── annotations.yaml
+│   └── Dockerfile
+```
+
+Build the bundle image inside of the version directory:
+```sh
+$ export USERNAME=<quay-username>
+$ export VERSION=<release-version>
+cd community-operators/koku-metrics-operator/$VERSION
+$ docker build -f Dockerfile . -t quay.io/$USERNAME/koku-metrics-operator-bundle:$VERSION
+```
```

Comment:
For testing, we don't need the community operators. We could just copy the new release to the testing folder, and do all of the image building there:
```
cp -r koku-metrics-operator/<release-version> testing
cd testing/<release-version>
$ docker build . -t quay.io/$USERNAME/koku-metrics-operator-bundle:$VERSION
$ docker push quay.io/$USERNAME/koku-metrics-operator-bundle:$VERSION
```

Then continue with the `opm` steps.

Reply:
I updated this to use the testing directory.